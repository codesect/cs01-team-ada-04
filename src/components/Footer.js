import React from 'react';
import styled from 'styled-components/macro';

import { Wrapper } from './GlobalStyles';

const StyledFooter = styled.footer`
  background-color: ${({ theme }) => theme.footerBackground};
  color: ${({ theme }) => theme.footerText};
  font-size: 0.75rem;
  text-align: center;

  p {
    margin: 0;
  }
`;

function Footer() {
  return (
    <StyledFooter>
      <Wrapper>
        <p>Made by Aimen Aounallah and Zsolt Meszaros for CodeSect</p>
      </Wrapper>
    </StyledFooter>
  );
}

export default Footer;
