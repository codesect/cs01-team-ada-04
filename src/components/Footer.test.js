import React from 'react';
import { render } from 'enzyme';
import 'jest-styled-components';

import Footer from './Footer';

describe('<Footer />', () => {
  it('matches snapshot', () => {
    const FooterWithTheme = withTheme(Footer);
    const wrapper = render(<FooterWithTheme />);

    expect(wrapper).toMatchSnapshot();
  });
});
