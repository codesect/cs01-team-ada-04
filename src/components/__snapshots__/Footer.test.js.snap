// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches snapshot 1`] = `
.c1 {
  margin: 0 auto;
  max-width: 832px;
  padding: 0.5rem 1rem;
}

.c0 {
  background-color: #8555c5;
  color: #fff;
  font-size: 0.75rem;
  text-align: center;
}

.c0 p {
  margin: 0;
}

<footer
  class="c0"
>
  <div
    class="c1"
  >
    <p>
      Made by Aimen Aounallah and Zsolt Meszaros for CodeSect
    </p>
  </div>
</footer>
`;
